docs/pages/push-notifications/fcm-credentials.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Add Google Service Account Keys using FCM V1'
+sidebar_title: Add Android FCM V1 credentials
+description: Learn how to create a Google Service Account Key for sending Android Notifications using FCM V1
+---
+
+import { Step } from '~/ui/components/Step';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Tab, Tabs } from '~/ui/components/Tabs';
+
+## Create a new Google Service Account Key
+
+Here are the steps to configure a new Google Service Account Key in EAS for sending Android Notifications using FCM V1.
+
+<Step label="1">
+  Create a new Firebase project for your app in the [Firebase Console](https://console.firebase.google.com). If you already have a Firebase project for your app, continue to the next step.
+
+  <ImageSpotlight
+    alt="Create a project in the Firebase Console"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/01-new-firebase-project.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+<Step label="2">
+  In the Firebase console, open Settings > [Service Accounts](https://console.firebase.google.com/project/_/settings/serviceaccounts/adminsdk) for your project.
+
+  <ImageSpotlight
+    alt="Firebase Console > Settings > Service Accounts"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/02-manage-service-accounts.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+<Step label="3">
+  Click **Generate New Private Key**, then confirm by clicking **Generate Key**. Securely store the JSON file containing the private key.
+
+  <ImageSpotlight
+    alt="Generate a new private key in the Firebase Console"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/03-generate-key.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+<Step label="4">
+  Upload the JSON file to EAS and configure it for sending Android notifications. This can be done in EAS CLI or via [expo.dev](https://expo.dev).
+
+  <Tabs tabs={["EAS CLI", "expo.dev"]}>
+    <Tab>
+      - Run `eas credentials`
+      - select `Android`
+      - select `production`
+      - select `Google Service Accounts`
+      - and select `FCM V1`
```

Comment:
```suggestion
      - Run `eas credentials`
      - Select `Android` > `production` > `Google Service Account`
      - Select `Manage your Google Service Account Key for Push Notifications (FCM V1)`
```

 - In CLI we call it "Google Service Account" instead of "Google Service Accounts". Let's use the same convention here. Similar to the last step.
 - Since we mention selecting `production` build profile here, what happens if I want to test the app first in development? Should I select the development profile? Should I keep separate Service Account keys for separate profiles in my project? I think addressing that in this guide or in the push notifications setup guide will be great to eliminate this confusion.

Reply:
@christopherwalter Do you have an answer to the second question here?

Reply:
> Since we mention selecting production build profile here, what happens if I want to test the app first in development?

Yeah good question, this is something I punted on and never circled back. I'm not sure if you can have different credentials for different build profiles (based on the entity structure on the backend, I'd assume build profile doesn't come into play). @quinlanj might be able to clarify this.

> In CLI we call it "Google Service Account" instead of "Google Service Accounts". Let's use the same convention here. Similar to the last step.

FWIW I think we should change all of these to "Google Service Account Keys" (the pluralization is important, as well as clarifying that these are Service Account *Keys* and not simply Service Accounts). With that said, I'm fine with fixing it here for consistency with the CLI.

Reply:
So it doesn't matter if you select Production, Development, etc. The GSAK will get stored the exact same way

Reply:
Okay so it seems like we can leave the doc here the way it is for now, and we should make another task to update places to use "keys" instead of "key".

Reply:
I've created an issue to fix this in the future https://linear.app/expo/issue/ENG-11650/eas-credentials-build-profile